src/lib/styles/dialogs.css

```
@@ -74,57 +137,102 @@
+/* The sheet presentation gives the close button a 44px touch target. */
+.dialog--sheet .dialog-close {
+  width: var(--touch-target-min, 44px);
+  height: var(--touch-target-min, 44px);
+}
+
+.dialog--sheet .dialog-content {
+  padding: var(--dialog-content-mobile-padding, var(--space-4));
+  padding-bottom: max(
+    var(--dialog-content-mobile-padding-bottom-min, var(--space-5)),
+    calc(
+      env(safe-area-inset-bottom, 0px) +
```

Stylelint (17.12.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)